generators/server/templates/src/main/java/package/config/WebConfigurer.java.ejs

```diff
@@ -215,13 +215,12 @@ public class WebConfigurer implements <% if (!reactive) { %>ServletContextInitia
     }
   <%_ } _%>
 
-  /*
     @Bean
     @Order(-2) // The handler must have precedence over WebFluxResponseStatusExceptionHandler and Spring Boot's ErrorWebExceptionHandler
```

Comment:
Is it still needed?

Reply:
Yes, this is required as sometimes exceptions happen prior to the controller selection that doesn't end up being caught by the default or extended handlers. This PR introduces [ReactiveWebExceptionHandler](https://github.com/dwarakaprasad/jhipster-bom/blob/fix_19991_spring_boot3_problem/jhipster-framework/src/main/java/tech/jhipster/web/rest/errors/ReactiveWebExceptionHandler.java) that replaces zalando's ProblemExceptionHandler.